pylint/checkers/spelling.py

```diff
@@ -16,8 +16,12 @@
 
 try:
     import enchant
+    from enchant.tokenize import get_tokenizer, Filter, EmailFilter, URLFilter, WikiWordFilter
 except ImportError:
     enchant = None
+    class Filter:
```

Comment:
About failure in travis - I can't see coverage data, but if there was an ImportError, Pylint would fail to infer `_skip` method in parent class when analyzing subclasses of Filter, and as a result it would treat it as new method, not as a overridden function.

Simply add interface method to your stub class:
```
class Filter:
    def _skip(self, word):
        raise NotImplementedError
```